rules/Php85/Rector/FuncCall/OrdSingleByteRector.php

```diff
@@ -67,6 +67,10 @@ public function refactor(Node $node): ?Node
         }
 
         $args = $node->getArgs();
+        if (! isset($args[0])) {
+            return null;
+        }
```

Comment:
This is not possible, as one arg is required: https://www.php.net/manual/en/function.ord.php

Reply:
I've read the other comment 👌